packages/webui/src/client/lib/viewPort.ts

```diff
@@ -192,36 +220,25 @@ async function innerScrollToSegment(
 
 		return scrollToPosition(top + window.scrollY, noAnimation).then(
 			async () => {
-				// retry scroll in case we have to load some data
-				if (pendingSecondStageScroll) window.cancelIdleCallback(pendingSecondStageScroll)
 				return new Promise<boolean>((resolve, reject) => {
-					// scrollToPosition will resolve after some time, at which point a new pendingSecondStageScroll may have been created
-
-					pendingSecondStageScroll = window.requestIdleCallback(
-						() => {
-							if (!secondStage) {
-								let { top, bottom } = elementToScrollTo.getBoundingClientRect()
-								top = Math.floor(top)
-								bottom = Math.floor(bottom)
-
-								if (bottom > Math.floor(window.innerHeight) || top < headerHeight) {
-									innerScrollToSegment(
-										elementToScrollTo,
-										forceScroll,
-										true,
-										true,
-										partInstanceId
-									).then(resolve, reject)
-								} else {
-									resolve(true)
-								}
+					if (!secondStage) {
+						//  Wait to settle 1 atemt to scroll
+						setTimeout(() => {
```

Comment:
I'm a bit worried that this timeout is never cleared, if `innerScrollToSegment` is called again, before the previous one is finished?

Reply:
Has added a cleanup